lib/http-server.js

```diff
@@ -54,7 +54,9 @@ function HttpServer(options) {
       ? 'html'
       : options.ext;
   }
-
+if(this.gzip) {
+    this.headers['Content-Encoding'] = 'gzip, deflate';
```

Comment:
you shouldn't need `deflate` here because we don't compress using the deflate algorithm.